os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -452,7 +452,7 @@ static int reloading_thread(int argc, char *argv[])
 	/* argv[1] binary index for reloading */
 	int bin_idx = (int)atoi(argv[1]);
 #ifdef CONFIG_SUPPORT_COMMON_BINARY
-	if (bin_idx == BM_BINID_LIBRARY) {
+	if (bin_idx == COMMLIB_IDX) {
```

Comment:
How about using `CMNLIB_IDX` instead of `COMMLIB_IDX`? We already used common as `cmn` in tcb.